### ISSUE TITLE
cannot find module ./types/bmp in a Meteor project

### ISSUE THREAD

**Issue by influxa** [image-size/image-size#81]
Hello. Trying to use this package in a Meteor js project. 

But I'm getting the error:
cannot find module ./types/bmp

How do I get this resolved? Many thanks in advance.

**Comment by charliesneath** [comment]
Having the same issue. @netroy it seems like the script is looking for the `type` folder in the wrong place (but the `types` folder does exist).

**Comment by zeke** [comment]
Can you share the following:

- your version of image-size
- your code
- your bmp file

**Comment by charliesneath** [comment]
Version: 0.5.4 (installed today).

I'm using it in the context of Framer.js

```
# npm.coffee
exports.sizeOf = require 'image-size'

# app.coffee
{sizeOf} = require 'npm'
```

I haven't been able to use the code yet as the error is happening on load.

**Comment by charliesneath** [comment]
I'm not sure if this is helpful, but looking at the build folder structure, I don't see the `types` folder: 

![image](https://cloud.githubusercontent.com/assets/606854/26263272/1c6912ba-3ca6-11e7-9893-5edc04eaf1e4.png)

**Comment by zeke** [comment]
Strange. I can see it:

```
❯ yarn add image-size --silent
warning No license field
└─ image-size@0.5.4
warning No license field

~/Desktop master*
❯ cat node_modules/image-size/package.json | json version
0.5.4

~/Desktop master*
❯ tree node_modules/image-size/lib 
node_modules/image-size/lib
├── detector.js
├── index.js
├── readUInt.js
├── types
│   ├── bmp.js
│   ├── dds.js
│   ├── gif.js
│   ├── jpg.js
│   ├── png.js
│   ├── psd.js
│   ├── svg.js
│   ├── tiff.js
│   └── webp.js
└── types.js

1 directory, 13 files
```

What happens if you `rm -rf node_modules && npm i image-size` ?

**Comment by charliesneath** [comment]
No change. I can see the entire directory you've pasted above on disk, but it seems like it's not being included in the node environment that Framer is creating. 

That screenshot I took is from Framer's inspector — is there any reason why this wouldn't include some folders? 

I made a sample repo that can reproduce the error. Understand that you might not be familiar with Framer (https://framer.com/) but it might help to see these files in action:

https://github.com/charliesneath/image-size-framer-test
